scripts/lightning.sh

```
@@ -0,0 +1,104 @@
+    lnd_fund_node bob_lnd
+  done
+
+  info "mining $fund_confirms blocks"
+  bitcoin_cli -generate $fund_confirms > /dev/null
+
+  lightning_sync
+}
+
+lnd_fund_node() {
+  local lnd_service_name=$1
+  local address=$(lncli $lnd_service_name newaddress p2wkh | jq -r .address)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)